<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageNewsDialog</class>
 <widget class="QDialog" name="ManageNewsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage News Subscriptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>News Types</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Create or add a custom news type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditTitle"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Header Bytes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditBytes"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonWrite">
     <property name="text">
      <string>Create Type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
